docs/content/distribute-bundles.md

```diff
@@ -90,7 +95,18 @@ porter publish -a mybunz1.1.tgz --tag getporter/megabundle:1.1.0
 
 ## Image References After Publishing
 
-When a bundle is published, the images that it will use are copied into the location of the published bundle. This simplifies access control and management of artifacts in the repository. Consider the following `porter.yaml` snippet:
+When a bundle is published, all images [referenced][image-map] by the bundle are
+published **inside** bundle repository. Bundles should be written to use the
+relocated image locations.
+
+* REGISTRY/ORG/BUNDLE:TAG
+  * REGISTRY/ORG/BUNDLE@**INVOCATION_IMAGE_DIGEST**
+  * REGISTRY/ORG/BUNDLE@**REFERENCED_IMAGE_1_DIGEST**
+  * REGISTRY/ORG/BUNDLE@**REFERENCED_IMAGE_2_DIGEST**
+
+NOTE: Digest refers to the the repository digest (not the image id).
```

Comment:
Maybe link [this section of the OCI specification](https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests)?